collections/Lyell/MS_Lyell_50.xml

```diff
@@ -48,25 +42,38 @@
                   <repository>Bodleian Library</repository>
                   <idno type="shelfmark">MS. Lyell 50</idno>
                </msIdentifier>
+               <head>Theological <foreign>quaestiones</foreign>; Gilbert Crispin; Honorius Augustodunensis</head>
```

Comment:
Add date and origin to `<head>`?

Reply:
I am using this field for the titles that DLM gives MSS (not including the date): should it be somewhere else?

Reply:
No, that is the right place. If there is a `head` element, it's used to generate the summary entry that you see in a page of search results; so in fact there's no harm in including date and origin as well.